### ISSUE TITLE
Curate / centralize meta-analytic stats

### ISSUE THREAD

**Issue by acganesh** [dcharrezt/NeurIPS-2019-Stats#4]
Kudos to @negrinho for noticing this.

There seem to be various discrepancies in meta-analysis stats across different conversations (e.g. on Twitter vs. from this repo vs. at @gradientpub).  It would be very nice to have some kind of central resource that maintains meta-analytic tooling and data so that we can all be on the same page.

I am happy to contribute to development here if helpful.

**Comment by dcharrezt** [comment]
Hi @acganesh,

Thanks to @negrinho for noticing it, I updated the repo. It would be great to have tooling to make analytics. For this repo I reused code from my previous post [ICML stats](https://github.com/dcharrezt/ICML-2019-Stats)

It is hard to match institutions since authors fill them manually, and there are problems like typos,

> Approximate Feature Collisions in Neural Nets
> Ke Li (UC Berkeley) · Tianhao Zhang (Nanjing University) · Jitendra Malik (**University of California at Berkley**)

Where 2 papers wrote berkley instead of berkeley.

**Comment by acganesh** [comment]
Hi @dcharrezt, thanks for the updates.  I'm working on a centralized meta-analytic repo here: https://github.com/gradientpub/meta-research

I'd really appreciate contributions if you want to modularize this code and make it usable across conferences.  I'm working on this in parallel, so let me know how you want to collaborate.
